src/Umbraco.Web.UI.Client/src/packages/core/temporary-file/temporary-file-manager.class.ts

```diff
@@ -78,6 +78,7 @@ export class UmbTemporaryFileManager<
 	}
 
 	async #validateItem(item: UploadableItem): Promise<boolean> {
+		await this.#temporaryFileConfigRepository.initialized;
```

Comment:
@iOvergaard Whilst testing, I'm hitting an issue here, the `initialized` doesn't resolve. But when I debug through the `UmbTemporaryFileConfigRepository` code, it looks like it should resolve. 😕

Reply:
I recall Niels mentioned something about changing consumeContext. We could get hit by something there with the whole resolution of promises. I'll have a look at it.

Also, on a side note, I don't care much about this `initialized` promise, but I couldn't see my way out of using that. Do you have an idea regarding this?